public/css/calendar.less

```diff
@@ -168,6 +178,50 @@
     border-left: none;
   }
 
+  .step {
+    grid-column-end: span @columnsPerDay;
+    grid-row-end: span @rowsPerHour;
+    position: relative;
+  }
+
+  .extra-count {
+    height: auto;
+  }
+}
+
+.extra-count {
+  z-index: 99;
+  background-color: @text-color-inverted;
+  border-radius: 0.25em;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  padding: 0 .25em;
+  color: @icinga-blue;
+}
```

Comment:
Please add this to the `.calendar-grid` ruleset. I don't like this being in the global scope.

Reply:
And also adjust it like this, since by making it a `ButtonLink` standard styles already apply.

```suggestion
.extra-count {
  z-index: 99;
  position: absolute;
  bottom: 1px; // 1px is the border-width of an entry, this makes it so it doesn't overlap
  right: 1px;
}
```

Reply:
It should look like this in the end:
![image](https://github.com/Icinga/icinga-notifications-web/assets/16668527/90df3ce9-4cab-400b-aa8f-8a9372b7fa84)